libs/foundry-dev-tools/src/foundry_dev_tools/errors/meta.py

```diff
@@ -57,7 +57,9 @@ def __init__(self, response: requests.Response | None = None, info: str | None =
             self.message = api_message
             del self.kwargs["message"]
         term_size = shutil.get_terminal_size().columns
-        msg = f"{self.message}\n{self.info}"
+        msg = self.message
+        if self.info:
+            msg += "\n{self.info}"
```

Comment:
What is the reason for outsourcing the `self.info` into a separate if-clause? Or was it just something you noticed that had to be fixed anyway?

Reply:
Ahh alright, I get that